### ISSUE TITLE
Accessibility: Video import module

### ISSUE THREAD

**Issue by kevinkidwell** [madebymany/sir-trevor-rails#64]
**Describe the issue**
This falls under the accessibility issue of "Inline frame without a text alternative." When users upload a video using the video module, the iFrame that is added does not meet WCAG 2.1 AA standards.

**Expected behavior**
Videos added using the video module should have the title attribute added (either automatically or manually by user).

**Related Issue**
[exhibits-library-cornell-edu#527](https://github.com/cul-it/exhibits-library-cornell-edu/issues/527)
